src/core/parser.ml

```diff
@@ -670,9 +676,8 @@ let parser cmd =
       -> P_rules(r::rs)
   | ms:modifier* _definition_ s:ident al:arg* ao:{":" term}? "≔" t:term
       -> P_definition(ms,false,s,al,ao,t)
-  | ms:modifier* _inductive_ i:ident t:{":" term} "≔"
-                 c:{ "|"  ilist:ident ":" tlist:term }*
-      -> P_inductive(ms, i, t, c)
+  | ms:modifier* _inductive_ i:inductive il:{ _:_with_ inductive}*
```

Comment:
remove `_:` before `_with_`

Reply:
I should do the same thing in the case _rule_?

Reply:
ok

Reply:
In fact, if I deleted this, I obtain an error:
```
Error: This expression has type
         (p_rule list -> p_rule -> 'a -> p_command_aux) Earley.grammar
       but an expression was expected of type
         ((unit * (p_term_aux loc * p_term_aux loc) loc) list -> 'b)
         Earley.grammar
       Type p_rule = (p_type * p_type) loc is not compatible with type
         unit * (p_term_aux loc * p_term_aux loc) loc 
```